notebooks/official/matching_engine/two-tower-model-introduction.ipynb

```diff
@@ -0,0 +1,1269 @@
+{
```

Comment:
<p>You'll need to fix the links so that they point to this notebook (after it gets merged).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>